frontend/src/types/home-gallery.ts

```diff
@@ -0,0 +1,2 @@
+const GALLERY_SETS = ["universe", "pottery", "olympics", "random"] as const
+export type GallerySet = (typeof GALLERY_SETS)[number]
```

Comment:
Curious: why extract this if it is only used in one place?

Reply:
The type is also used in the extracted `VGalleryImage` component, and you can't export types from `script setup`